__test__/backend.test.js

```diff
@@ -222,7 +214,7 @@ it('should create campaign contacts', async () => {
 
   const contact = await createContact(campaign.data.createCampaign.id)
 
-  expect (contact.campaign_id).toBe(campaign.data.createCampaign.id)
+  expect (contact.campaign_id).toBe(parseInt(campaign.data.createCampaign.id))
```

Comment:
This is a current wart with the current graphQl 'spec' and our backend model.  I'm scared to tackle changing it all the way up/down the api and frontend.  someday....